src/cli/cli_install.rs

```diff
@@ -94,7 +94,7 @@ pub struct Settings {
 fn print_long_description(settings: &Settings) {
     println!();
     print_markdown!("\
-        # Welcome to EdgeDB!\n\
```

Comment:
This is interesting. Does it have a better style? We should probably change the style instead?

Reply:
Found a way to fix the alignment.